docs/blog.md

```diff
@@ -70,7 +70,7 @@ This allows us to express the following: "If someone is interacting via the keyb
 
 Rules like this would allow users of touch devices or mouse oriented users to avoid the focus ring except in cases where the only available or very likely input modality was a keyboard (like `<input type=text>`), while allowing keyboard users to see the focus ring on anything with focus based on the fact that they are using the keyboard. If a user begins using the mouse, simply pressing tab/shift tab will let them know where they are by shifting the modality back to keyboard.
 
-We've created a rough prollyfill for this concept. Because we can't (yet) polyfill media queries, it uses the more basic mechanism on setting an attribute on `<body>`, so the above rule would look more like
+We've created a rough prollyfill for this concept. Because we can't (yet) prolyfill media queries, it uses the more basic mechanism on setting an attribute on `<body>`, so the above rule would look more like
```

Comment:
On Sat, Jul 25, 2015 at 1:33 PM, Alice <notifications@github.com> wrote:

> In docs/blog.md
> <https://github.com/bkardell/modality/pull/4#discussion_r35484966>:
>
> > @@ -70,7 +70,7 @@ This allows us to express the following: "If someone is interacting via the keyb
> >
> >  Rules like this would allow users of touch devices or mouse oriented users to avoid the focus ring except in cases where the only available or very likely input modality was a keyboard (like `<input type=text>`), while allowing keyboard users to see the focus ring on anything with focus based on the fact that they are using the keyboard. If a user begins using the mouse, simply pressing tab/shift tab will let them know where they are by shifting the modality back to keyboard.
> >
> > -We've created a rough prollyfill for this concept. Because we can't (yet) polyfill media queries, it uses the more basic mechanism on setting an attribute on `<body>`, so the above rule would look more like
> > +We've created a rough prollyfill for this concept. Because we can't (yet) prolyfill media queries, it uses the more basic mechanism on setting an attribute on `<body>`, so the above rule would look more like
>
> nah, leave it as polyfill the second time - it's the generic term, and
> here it's irrelevant whether we're talking about a nascent spec or
> something which exists in all but one browser; it's the capability to do
> either which doesn't exist.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bkardell/modality/pull/4/files#r35484966>.
>

yeah, ok, I vacillated on this myself.

-- 
Brian Kardell :: @briankardell :: hitchjs.com